omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -87,10 +86,27 @@
 	
 	private Logger log = Logger.getLogger(this.getClass());
 	
-	public SwaggerSpecificationCreator(String baseUrl) {
-		this.baseUrl = baseUrl;
```

Comment:
Is removing the lines below related to? "Correct host property in OpenAPI Spec"

Reply:
We no longer use the baseUrl constructor. However the `searchHandlerDocs` instance variable requires baseUrl param to initiate. `searchHandlerDocs` is only referenced by getParametersListForSearchHandlers() which is unused. So it makes sense to remove these here.

Reply:
I do not refute the fact that we no longer need some code. What am simply saying is that if removing it is not related to "Correct host property in OpenAPI Spec", then it should be in a separate pull request.